drag_and_drop_v2/public/js/view.js

```diff
@@ -50,16 +51,28 @@
         if (item.is_placed) {
             style.left = item.x_percent + "%";
             style.top = item.y_percent + "%";
+            tabindex = -1;  // If an item has been placed it can no longer be interacted with,
+                            // so remove the ability to move focus to it using the keyboard
+        }
+        var content_html = item.displayName;
+        if (item.imageURL) {
+            content_html = '<img src="' + item.imageURL + '" alt="' + item.imageDescription + '" />';
```

Comment:
@mtyaka Thanks for the update. We do support HTML inside `displayName`, see [test_html_data.json](https://github.com/open-craft/xblock-drag-and-drop-v2/blob/a11y/tests/integration/data/test_html_data.json#L24).